src/ToggleSwitch/ToggleSwitch.features.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import ToggleSwitch from './ToggleSwitch'
+import {Box, Text} from '..'
+import {action} from '@storybook/addon-actions'
+
+export default {
+  title: 'Components/ToggleSwitch/Features',
+}
+
+export const Small = () => (
+  <Box
+    display="grid"
+    gridTemplateColumns={'max-content auto'}
```

Comment:
Nitpick: You can leave out the curly braces if the prop is a string literal

```suggestion
    gridTemplateColumns="max-content auto"
```